### ISSUE TITLE
Twitter.jl on a 32-bit  platform (Raspberry Pi)

### ISSUE THREAD

**Issue by Alexander-Barth** [randyzwitch/Twitter.jl#22]
Thanks for this nice package!

I try to use Twitter.jl on a Rasberry Pi, but I have the following error (while it works on a 64-bit Linux):

```julia
Twitter.post_status_update(status = "Test")
# returns
# ERROR: InexactError: trunc(Int32, 1180186675730898944)
# Stacktrace:
# [1] throw_inexacterror(::Symbol, ::Any, ::Int64) at ./boot.jl:567
```

On 32-bit platforms, `Int` has just 32-bits. I propose to use actually `Int64` for all IDs. 
See also https://blog.twitter.com/developer/en_us/a/2013/64-bit-twitter-user-idpocalypse.html

I will submit a PR.

I am using Julia 1.0.3.

**Comment by randyzwitch** [comment]
Fixed by #23

**Comment by Alexander-Barth** [comment]
So fast! Thanks a lot!
